2/contrib/openshift/base-plugins.txt

```diff
@@ -14,7 +14,7 @@ groovy:2.4
 htmlpublisher:1.25
 jira:3.7.1
 job-dsl:1.77
-junit:1.52
+junit:1119.1121.vc43d0fc45561
```

Comment:
Why do we end up going back and forth between versions such as `1.52` and `1119.xxxx.xxxx.xxxx`?

Reply:
This version was required by another plugin (workflow-api:1144.v61c3180fa_03f iirc)